.github/workflows/release.yml

```diff
@@ -14,8 +14,8 @@ jobs:
   main:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
```

Comment:
`v4` is the latest major version of both of these actions, at the time of this writing.